apps/docs-snippets/src/guide/contracts/multicalls.test.ts

```diff
@@ -33,7 +33,9 @@ describe(__filename, () => {
     );
 
     echoContract = await factory1.deployContract();
-    counterContract = await factory2.deployContract();
+    counterContract = await factory2.deployContract({
```

Comment:
I think we should change the counter contract and remove the try_read and the unwrap_or to make sure the initialization worked.

Reply:
@luizstacio I removed this from the contract and now some new unrelated test that uses this contract started failing haha

Reply:
It passes if I pass in the storage slots imported from the storage slots json file

Reply:
Is this expected behaviour? Should it fail without proper initialization when deploying the contract?

Reply:
You managed to fix it, right? I see the tests are passing now.

Can we merge and release it via [#1232](https://github.com/FuelLabs/fuels-ts/pull/1232)?

Reply:
@Dhaiwat10 yeah is expected that the other tests fails